### PR TITLE
Fixed #25376 -- Required virtualenv in installation instructions.

### PR DESCRIPTION
**Pull request by timgraham**
https://code.djangoproject.com/ticket/25376

### CODE REVIEW

**Comment by collinanderson** [comment]
oooh wow. Those contributing docs on virtualenv are really good. I wonder if we should just move them here. Maybe for another day... :)

File: docs/topics/install.txt [hunk 1]
```diff
@@ -253,101 +214,31 @@ latest bug fixes and improvements, follow these instructions:
...
 
-        $ git clone git://github.com/django/django.git django-trunk
+        $ git clone git://github.com/django/django.git
 
-   This will create a directory ``django-trunk`` in your current directory.
+   This will create a directory ``django`` in your current directory.
 
 3. Make sure that the Python interpreter can load Django's code. The most
-   convenient way to do this is via pip_. Run the following command:
+   convenient way to do this is to use `virtualenv`_ and pip_. The
```
Comment (berkerpeksag) [inline docs/topics/install.txt]: \`virtualenv\`\_ -> virtualenv\_

**Comment by FlipperPA** [comment]
By way of background, I have to support an organization of developers in a BYOD environment. Some have Windows, some have Macs, some have Ubuntu, and some develop directly on RedHat EL6 servers. Some use Vagrant, some do not. I say this to recommend not just virutalenv but virutalenvwrapper: not only are the commands easier / straightforward for a newcomer, virtualenvwrapper takes a LOT of pain away from Windows developers. Otherwise, they seem not need a Masters degree in Python pathing. :)

**Comment by timgraham** [comment]
I agree, Tim. It's mentioned once, but not in "installing the development version section" so I added it there. Also, there is also `docs/howto/windows.txt` -- if you have some colleagues who use Windows regularly, perhaps they would like to contribute their best practices.

**Comment by FlipperPA** [comment]
I started working on this, and have written up a page with instructions, including virtualenv / virtualenvwrapper-win as an optional step. This was my first time installing 3.5 on Windows, and I was pleasantly surprised.

The installation process on Windows is much more in line with typical Windows installs, and includes an option to include Python & pip in the Windows path with a checkbox from the wizard.

Are we okay with the instructions being written for Python 3.5 on Windows? This would be forward looking, with Django 1.9 dropping support for Python 3.3. Unfortunately, the Python 2.7.10 install still uses the old C:\Python27 path and doesn't give the option for including it in the Windows path, so we're dealing with divergent Windows install wizards.

**Comment by timgraham** [comment]
Yes, that's fine with me.

**Comment by FlipperPA** [comment]
PR issued. Thanks Tim, let me know what you think.

**Pull request merged**